srv/salt/ceph/openattic/default.sls

```diff
@@ -5,3 +5,13 @@ install openattic:
     - pkgs:
       - openattic
 
+enable openattic-rpcd:
```

Comment:
The automatic start/enablement was removed from the SES RPM spec file, as this is against the packaging guidelines. We're looking into adding this to `oaconfig` instead, e.g. creating `oaconfig enable` that will make sure that all required services (Apache, Nagios, PostgreSQL) are enabled upon reboot.